packages/esm-patient-chart-app/src/visit/visits-widget/past-visits-components/medications-summary.component.tsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import capitalize from 'lodash-es/capitalize';
+import dayjs from 'dayjs';
+import { useTranslation } from 'react-i18next';
+import { OrderItem, getDosage } from '../visit.resource';
+import styles from '../visit-detail-overview.scss';
+
+function formatTime(dateTime) {
+  return dayjs(dateTime).format('hh:mm');
+}
+
+interface MedicationSummaryProps {
+  medications: Array<OrderItem>;
+}
+
+const MedicationSummary: React.FC<MedicationSummaryProps> = ({ medications }) => {
+  const { t } = useTranslation();
+
+  return (
+    <React.Fragment>
```

Comment:
Personally, I prefer the shorter `<>` in case no props etc. must be given.

But that's a nitpick and you can leave it as is.